src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -20,29 +19,20 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        internal int _version;
         private List<Cookie> _list = new List<Cookie>();
 
         private DateTime _timeStamp = DateTime.MinValue;
         private bool _hasOtherVersions;
 
-        private readonly bool _isReadOnly;
-
         public CookieCollection()
         {
-            _isReadOnly = true;
-        }
-
-        internal CookieCollection(bool IsReadOnly)
```

Comment:
Fair enough. I will keep this in mind going forward.

In this particular case, I'd be very surprised if it's being used in .NET Native, unless .NET Native has a completely different implementation of `CookieCollection` than CoreFx/RefSource that actually respects `_isReadOnly`.